src/connection_properties.lua

```diff
@@ -29,8 +30,43 @@ function ConnectionProperties:_validate()
     end
 end
 
+--- Get the configured fetchsize in bytes used when fetching query result data.
+--- Configuration property: <code>fetchsize_kib</code>.
+--- Default value: <code>131072</code> = <code>128*1024</code>.
+--- @return number fetchsize in bytes
 function ConnectionProperties:get_fetchsize_bytes() --
     return (self.properties.fetchsize_kib or DEFAULT_FETCHSIZE_KIB) * 1024
 end
 
+--- Get the configured TLS verify mode for connecting to Exasol.
+--- Configuration property: <code>tls_verify</code>.
+--- Default value: <code>none</code>.
+--- Available values: <code>none</code>, <code>peer</code>, <code>client_once</code>, <code>fail_if_no_peer_cert</code>.
+--- See https://github.com/brunoos/luasec/wiki/LuaSec-1.1.0#sslnewcontextparams
+--- @return string TLS verify mode
+function ConnectionProperties:get_tls_verify() --
+    return self.properties.tls_verify or "none"
+end
+
+--- Get the configured TLS protocol for connecting to Exasol.
+--- Configuration property: <code>tls_protocol</code>.
+--- Default value: <code>tlsv1_2</code>
+--- Available values: <code>tlsv1</code>, <code>tlsv1_1</code>, <code>tlsv1_2</code>, <code>tlsv1_3</code>.
+--- Note: Exasol does not yet support <code>tlsv1_3</code>.
```

Comment:
```suggestion
--- Note: Exasol does not yet support <code>tlsv1_3</code>.
```
If you put something like this in the code, you need a frame of reference, so say which is the version you are talking about. This comment does not age well.